doc.go

```diff
@@ -1,13 +1,40 @@
-// Package jsonnext provides some types and functions to make working with
-// jsonnet (github.com/google/go-jsonnet) easier and more featureful.
+// Package jsonnext makes it simpler to use jsonnet in a program.
+//
+// jsonnet is a data templating language that can be considered an extension of
+// JSON that makes it composable.
+//
+//  import "github.com/google/go-jsonnet"
+//
+// jsonnext provides some go types and functions to make it easier to plug
+// jsonnet into a program with minimal work and to extend some of the jsonnet
+// abstractions to add capability.
 //
 // Importer
 //
-// An implementation of jsonnet.Importer that supports netpaths - paths that
-// start with a double-slash - // - for network paths. Netpaths are retrieved
-// via https using a URLFetcher. Other paths are read from the filesystem.
+// An Importer is implementation of jsonnet.Importer that supports netpaths -
```

Comment:
I said "Importer is an implementation of ..." (move the "an", since it was missing in the second place and not needed in the first).

I didn't go with your wording because I wanted to say "that supports netpaths", and I can't see how to word that otherwise. "An Importer implements jsonnet.Importer supporting netpaths - " doesn't sound quite right to me.

Reply:
done-ish